<launch>

 <!-- PS3 joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="PS3_joy_node" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12"/>
  </node>


  <node pkg="rover1" type="user_input" name="user_input_node" output="screen"/>

  <node pkg="rover1" type="central_control" name="central_control_node" output="screen"/>

  <node pkg="rover1" type="drive_serial" name="drive_serial_node"
  output="screen"/>

  <node pkg="rover1" type="arm_serial" name="arm_serial_node"
  output="screen"/>
  
  <!--
  <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    <param name="port" value="/dev/ttyACM1"/>
    <param name="baud" value="115200"/>
  </node>

  <node pkg="ps3joy" type="ps3joy.py" name="ps3joy"/> 
  -->


</launch>
